AUTHORS

```diff
@@ -1,17 +1,36 @@
-Maintainer:
+Maintainers:
 
 * Franklin "Snaipe" Mathieu <franklinmathieu@gmail.com>
+* László "MrAnno" Várady <laszlo.varady@balabit.com>
```

Comment:
Wow, thank you for listing me!

Can I use my personal email address (laszlo.varady at protonmail.com)?

Reply:
Of course :slightly_smiling_face: